singularity/start.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# strict mode so script quits immediately upon failure
+set -eo pipefail
+
+shellmode=false
+
+while getopts ":s" opt; do
+    case $opt in
+        s)
+            shellmode=true
+            echo "Starting Singularity shell..."
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG"
+            ;;
+    esac
+done
+
+# Getting environment variables from .env
+source ./loadenv.sh
+source ./varcheck.sh
+
+if [ $shellmode = true ]
+then
+    singularity shell -B ${DPDASH_STATE_DIR}:/data -B ${DPDASH_DATA_DIR}:/project_data ${DPDASH_IMG}
```

Comment:
That makes sense. Entering and exiting the container can be done with the `-s` flag, but I get what you mean.

Would it be okay to keep these scripts intact but also give the full `singularity` commands in documentation? I only ask because these scripts have made my life simpler without having to write the full `singularity` commands with all of the mounts and everything. I could put them in a little side note in the documentation to be used at anyone's convenience.

Reply:
I've done that in https://github.com/PREDICT-DPACC/dpdash/pull/14/commits/590929f2c86cce76a18e9a3dbcb3de78a80ceacf. Let me know if you'd instead like me to just get rid of those `start.sh` and `stop.sh` scripts entirely and I can just keep them on my local machine. Just figured it couldn't hurt to provide them to other devs.

Reply:
Thanks for covering all bases!